internal/engine/upper_test.go

```diff
@@ -151,18 +151,10 @@ func TestUpper_UpWatchError(t *testing.T) {
 		f.watcher.errors <- errors.New("bazquu")
 	}()
 	err := f.upper.CreateManifests(output.CtxForTest(), []model.Manifest{manifest}, true)
-	close(f.b.calls)
 
 	if assert.NotNil(t, err) {
 		assert.Equal(t, "bazquu", err.Error())
 	}
-
-	var startedManifests []model.Manifest
```

Comment:
why did this test change?

Reply:
before, we would always run a set of builds before pulling from the file watching chans

now it's all in one select loop, so it's possible we get the watch error before BaD is invoked (`dispatch` is always invoked before we pull from the chan, but then it calls BaD in a separate goroutine).